pkgs/tools/admin/wander/default.nix

```diff
@@ -0,0 +1,22 @@
+{ buildGoModule, fetchFromGitHub, lib }:
+
+buildGoModule rec {
+  pname = "wander";
+  version = "0.4.1";
+
+  src = fetchFromGitHub {
+    owner = "robinovitch61";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-Z3jvKBhzlATTh6zPoJoMmg/DAE5/Ur3Tb3sdgGPEm6k=";
+  };
+
+  vendorSha256 = "sha256-gWQ8GbtghhCRq6tOU6qmWBuponmfUkUDAk3+dPtmMiE=";
+
+  meta = with lib; {
+    description = "Terminal app/TUI for HashiCorp Nomad";
+    license = licenses.mit;
+    homepage = "https://github.com/robinovitch61/wander";
+    maintainers = teams.c3d2.members;
```

Comment:
has everyone on this team agreed to be a maintainer for this?

Reply:
considering we want to use nomad at some point for our infrastructure I would say yes.

Reply:
> has everyone on this team agreed to be a maintainer for this?

The c3d2 team is for the hackspace with the same name and collects all software we package for our infrastructure. Before the team was created we kept packages under several different personal accounts which made it harder to keep track what we need to maintain collectively. It's fine for me (and probably the others) to add new packages that are/will be used in our infrastructure and I don't think we need the approval of each person in the team each time a new package is added.